src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
Concatenation is just a way of string interpolation. So instead of doing something like `"Note: ". $msg . " something else"` the logger can be called with a variable `message = value` and the log string would be `Note: {message} something else`